### ISSUE TITLE
New Rule : rule to verify bad code indentation 

### ISSUE THREAD

**Issue by AngelHely** [guillep/pharo-core#16698]
This could suggest verifying bad code indentation. For example :

```
methodName  | ...temps... | ...code... 
``` 

```
methodName 

    | ...temps... | ...code... 
```

```
methodeName 

    <pragma> ...code...
```

**Comment by jbrichau** [comment]
Yes, For your information, there are some additional code formatting rules in https://github.com/SeasideSt/PharoWithStyle 
These were extracted from Seaside-Slime mostly but I try to add new ones. Let's try for this indentation.

<blockquote><img src="https://opengraph.githubassets.com/9c57bcb34c54a36448ce98efddd294e4e660249b7f4058de157a91160a1b3f44/SeasideSt/PharoWithStyle" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/SeasideSt/PharoWithStyle">GitHub - SeasideSt/PharoWithStyle: Code style rules from the Pharo With Style book</a></strong></div><div>Code style rules from the Pharo With Style book. Contribute to SeasideSt/PharoWithStyle development by creating an account on GitHub.</div></blockquote>
